src/components/Canvas/ImageElement.vue

```diff
@@ -59,11 +62,14 @@ import {
 export default {
   watch: {
     imageId: function(newVal) {
-      this.shouldRunAnimation = false;
+      clearTimeout(this.timeoutEvent);
+      this.shouldDisapear = false;
```

Comment:
disappear has 2 p

Reply:
Remove, this attribute never used in the current implementation